packages/blocks/src/api/parser.js

```diff
@@ -461,7 +461,11 @@ export function createBlockWithFallback( blockNode ) {
 	// provided source value with the serialized output before there are any modifications to
 	// the block. When both match, the block is marked as valid.
 	if ( ! isFallbackBlock ) {
-		block.isValid = isValidBlockContent( blockType, block.attributes, innerHTML );
+		const parsedHTML = parseWithAttributeSchema( innerHTML, {
```

Comment:
This will degrade performance significantly for longer posts, given the implementation of `parseWithAttributeSchema`.